api/server/agent.go

```diff
@@ -32,21 +36,22 @@ func (tapi ToDDApi) Agent(w http.ResponseWriter, r *http.Request) {
 		// Make sure UUID string actually contains something
 		if len(uuid[0]) > 0 {
 			// Let's get the full list so we can identify the right agent if the user specified a short
-			full_agent_list := tdb.DatabasePackage.GetAgents()
+			full_agent_list, _ := tapi.tdb.GetAgents()
 
 			for i := range full_agent_list {
 				if strings.HasPrefix(full_agent_list[i].Uuid, uuid[0]) {
-					agent_list = append(agent_list, tdb.DatabasePackage.GetAgent(full_agent_list[i].Uuid))
+					agent, _ := tapi.tdb.GetAgent(full_agent_list[i].Uuid)
+					agent_list = append(agent_list, *agent)
```

Comment:
See my note at https://github.com/Mierdin/todd/pull/2/files#r57103867 - I am curious about the reasoning for returning a pointer here, and why not the same approach for GetAgents()